e2e/tests/api-driven/src/permissions/helpers.ts

```diff
@@ -24,26 +28,27 @@ export const cleanup = async () => {
 };
 
 export const setup = async () => {
+  const user1Id = await createUser({ email: "e2e-user-1@opensystemslab.io" });
   const teamId1 = await createTeam({ name: "E2E Team 1", slug: "e2e-team1" });
+  const team1FlowId = await createFlow({
+    teamId: teamId1,
+    slug: "team-1-flow",
+  });
+
+  const user2Id = await createUser({ email: "e2e-user-2@opensystemslab.io" });
   const teamId2 = await createTeam({ name: "E2E Team 2", slug: "e2e-team2" });
```

Comment:
double checking i understand this: in overall setup, we create a user who is by default _not_ a platform admin and they also do not have roles on either `teamId1` or `teamId2`? then later in individual test suites, you're creating & cleaning up team memberships and toggling permissions?

i'm wondering a bit if it'd be clearer to setup more explicitly with something like `e2e-platform-admin@..` etc rather than vague `user-1` etc? definitely not a showstopper though.

Reply:
Yeah I appreciate this is an unclear point - team1 and team2 are quite vague/meaningless names but I needed something to differentiate (I was also considering red/blue or something to make it slightly more distinct).

The intention is that there are -
 - Two teams
 - One TeamEditor of each team
 - We test that teamEditor1 can access team1 resources
 - We test that teamEditor2 cannot access team1 resources

I'll add comments to explicitly explain the setup here 👍